src/components/SongList/ListItemNodes.tsx

```diff
@@ -13,15 +13,15 @@ interface TitleProps {
 const TitleNode: React.FC<TitleProps> = ({ musicItem }) => {
   const [metadata, setMetaData] = useState<MusicNftMetaData>();
 
-  const fetchMetaData = async () => {
+  const fetchMetaData = useCallback(async () => {
```

Comment:
Don't you think optimizing `fetchMetadata` func with a callback is a bit of an overkill? Seeing as the TitleNode doesn't manage any internal states for it to cause a re-render?

Reply:
I don't think that will lead to performance issue and the rerendering by any parent node will not regenerate this function I believe so its useful than overkill, we can always address this again if we found any issues with it, right now I just followed lint issues

Reply:
Alright no problem